mozilla/foundation.mozilla.org
# added apple pay domain association view

# Description

Related PRs/issues: #9747

For the foundation stack, we need to add two Apple Pay Verification files to the /.well-known directory in order to enable Apple Pay for foundation.mozilla.org and mozillafestival.org.

It has been requested to take the contents of these 2 files provided by Fundraise Up, and save them as environment variables. Then using logic in the backend, serve the contents of the associated files when Apple Pay checks
"(base url)/.well-known/apple-developer-merchantid-domain-association"


**This PR adds:**
- Two new environment variables: 
  - `APPLE_PAY_DOMAIN_ASSOCIATION_KEY_FOUNDATION` 
  - `APPLE_PAY_DOMAIN_ASSOCIATION_KEY_MOZFEST`
 - A new view `apple_pay_domain_association_view`, which is called when Apple Pay or a user visits the route `/.well-known/apple-developer-merchantid-domain-association`. This view checks whether the request is being made from "foundation.mozilla.org" or "mozillafestival.org" and will serve what is set for the associated environment variable.

# Steps to test

1. Check this branch out
2. In your .env file, add string values for `APPLE_PAY_DOMAIN_ASSOCIATION_KEY_FOUNDATION`, `APPLE_PAY_DOMAIN_ASSOCIATION_KEY_MOZFEST`. (EX: "test_value_123")
3. Visit http://localhost:8000/.well-known/apple-developer-merchantid-domain-association, and the value you set for `APPLE_PAY_DOMAIN_ASSOCIATION_KEY_FOUNDATION` should appear.

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**
